src/Umbraco.Web/Net/AspNetHttpContextAccessor.cs

```diff
@@ -10,7 +10,13 @@ public HttpContextBase HttpContext
             get
             {
                 var httpContext = System.Web.HttpContext.Current;
-                return httpContext is null ? null : new HttpContextWrapper(httpContext);
+
+                if (httpContext is null)
```

Comment:
I wonder if this goes against the 'accessor' pattern? I think most accessors will just return null if there isn't one which also provides a way to determine if there is an object at the current state. By throwing here it means you can't check if there isn't one without a try/catch. I know that means we'd have to put null checks everywhere throughout the code.... else does it make sense to have an ext method for this like `httpContextAccessor.GetRequired()` which has this null check?

Reply:
I like the idea of an extension method.. I will introduce that